IBX-10685: Added information on how to add a custom attribute type to an existing storage definition (#3068)
ezsystems/developer-documentation: docs/pim/create_custom_attribute_type.md

### Storage definition

You can either create a new storage definition or use an existing one.

> To create a new one, prepare a `PercentStorageDefinition` class, which implements `Ibexa\Contracts\ProductCatalog\Local\Attribute\StorageDefinitionInterface`.

Reviewer: ```suggestion
To create a new storage definition, prepare a `PercentStorageDefinition` class, which implements `Ibexa\Contracts\ProductCatalog\Local\Attribute\StorageDefinitionInterface`.
```

[[= include_file('code_samples/catalog/custom_attribute_type/config/custom_services.yaml', 41, 44) =]]
```

> However, if you prefer to use an existing storage definition, you need to create a Storage Definition Tag CompilerPass `src/DependencyInjection/AddFloatStorageDefinitionTag.php`:

Reviewer: ```suggestion
If you prefer to use an existing storage definition, you need to create a Storage Definition Tag CompilerPass `src/DependencyInjection/AddFloatStorageDefinitionTag.php`:
```

``` php
[[= include_file('code_samples/catalog/custom_attribute_type/src/DependencyInjection/AddFloatStorageDefinitionTag.php') =]]
```

> Add the CompilerPass to the container. Do it in a `src/Kernel.php` file or in your Bundle class:

Reviewer: ```suggestion
Add the CompilerPass to the container.
Do it in a `src/Kernel.php` file or in your Bundle class:
```

@mateuszdebinski Thank you for directly contributing to the documentation!

> Add the CompilerPass to the container with a priority higher than 0 so that it runs first `src/Kernel.php`:

Reviewer: ```suggestion
Add the CompilerPass to the container with a priority higher than 0 so that it runs first.
To do it, create a `src/Kernel.php` file:
```

or...

"Do it in a `src/Kernel.php` file:"
